neovide-git.json

```diff
@@ -0,0 +1,6 @@
+{
+  "type": "git",
+  "url": "https://github.com/sqwxl/neovide.git",
+  "branch": "flatpak",
```

Comment:
NEVER use git branches.

Also I will re-iterate: inline this in the manifest, it's unmaintainable otherwise.